site/_posts/2022-11-01-authenticators-ceremonies-webauthn-oh-my.md

```diff
@@ -0,0 +1,128 @@
+
+## Why You Need WebAuthn
+
+You might be wondering why you should use WebAuthn instead of using a more robust password from a password generator. Or you may be considering adding 2FA for more security, which is an improvement but still prone to cyberattacks (*ie* phishing) [compared to WebAuthn](https://www.slashid.dev/blog/webauthn-antiphishing#:~:text=WebAuthn%20Anti%2DPhishing,superior%20authentication%20method.).
+
+Here are a few of the reasons WebAuthn is the superior choice for keeping your application secure:
+
+* WebAuthn helps resolve significant security problems related to data breaches, phishing, and attacks against 2FA methods. Phishing attacks are usually disguised emails or messages luring users to click on malicious links with the intent of stealing sensitive data like login credentials. With WebAuthn, attackers can't gain access to your data using login credentials stolen from your device since WebAuthn uses cryptographic keys to authenticate users and the private key never leaves the device nor is accessible to the user.
+* It's easier to use since users have to manage fewer passwords.
+* It eliminates the need for servers to store passwords. With passwords, if a server happens to be compromised, the user might be at risk if the password wasn’t hashed properly. With WebAuthn, if the server is compromised, the server only stores the public key, which requires the private key (stored in the user's device) to complete the authentication.
+* It's easier for attackers to steal a user's password than to steal their device. Even if an attacker does steal a device, they still need to bypass the device's security (*ie* biometrics or facial recognition system).
+* WebAuthn is easy to implement because it consists of [browser APIs that can be easily used by developers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API#examples).
+* It offers a smooth authentication experience for users.
+
+## How WebAuthn Works
+
+WebAuthn consists of three components: the authenticator (the user's device), the web browser, and the server (RP). In essence, the user initiates login via the browser, the browser then prompts the user to authenticate using their device (via biometrics or facial recognition, for example), and then if successful, the user is logged into the web application.
+
+However, this is only a basic view of the process. To understand the process in more depth, you need to look at the two main flows of WebAuthn: the registration flow and the authentication flow. (They are called "ceremonies" in the specification.)
+
+### The Registration Flow
+
+Before users can authenticate, their device(s) must be registered on the server of the RP. This way, the RP creates the credentials that the user can authenticate, and the process looks like this:
```

Comment:
```suggestion
Before users can authenticate, their device(s) must be registered on the server of the RP. This way, the RP creates the credentials that the user can use to authenticate, and the process looks like this:
```

It seems like there's a word or two missing from that line. Should it be something like "user can authenticate with" or "creates the authentication credentials for the user"?